examples/plasticity/WellboreProblem_PoroElastic.xml

```diff
@@ -0,0 +1,281 @@
+<!--
+Problem: wellbore problem subjected to mixed mode loading and anisotropic far-field stresses
+Created by: Jian Huang
+Last update: July 23 2021
+Objective: to verify the poroelastic solver in GEOSX against analytical solution
+Reference: Cheng, A. H. D. (2016). Poroelasticity (Vol. 877). Switzerland: Springer International Publishing.
+-->
+
+
+<Problem>
+<!-- SPHINX_WELLBORE_TABLE -->
+<Functions>
+  <TableFunction 
+    name="timeFunction"
+    inputVarNames="{time}"
+    coordinates="{0.0, 0.1, 1e6}"
+    values="{0.0, 1.0, 1.0}" 
+  />
+</Functions>
+<!-- SPHINX_WELLBORE_TABLE_END -->
+
+<!-- SPHINX_WELLBORE_MESH -->
+<Mesh>
+  <InternalWellbore
+    name="mesh1"
+    elementTypes="{ C3D8 }"
+    radius="{ 0.1, 5.0 }"
+    theta="{ 0, 180 }"
+    zCoords="{ -1, 1 }"
+    nr="{ 40 }"
+    nt="{ 80 }"
+    nz="{ 2 }"
+    trajectory="{ { 0.0, 0.0, -1.0 },
+                  { 0.0, 0.0,  1.0 } }"
+    autoSpaceRadialElems="{ 1 }"
+    useCartesianOuterBoundary="0"
+    cellBlockNames="{ cb1 }"/>
+</Mesh> 
+<!-- SPHINX_WELLBORE_MESH_END -->
+
+<!-- SPHINX_WELLBORE_MATERIAL -->
+<Constitutive>
+  <PorousElasticIsotropic
+    name="porousRock"
+    solidModelName="rock"
+    porosityModelName="rockPorosity"
+    permeabilityModelName="rockPerm"
+  />
+  <ElasticIsotropic
+    name="rock"
+    defaultDensity="2700"
+    defaultBulkModulus="11111111111.1"
```

Comment:
Use scientific notation for readability (1.111e10).  So many digits of accuracy are also unnecessary in practice :-)

Reply:
agree